graphql/schema.graphql

```diff
@@ -83,7 +83,7 @@ type Mutation {
   schedulePatch(patchId: String!, configure: PatchConfigure!): Patch!
   schedulePatchTasks(patchId: String!): String
   unschedulePatchTasks(patchId: String!, abort: Boolean!): String
-  restartPatch(patchId: String!, abort: Boolean!, taskIds: [String!]!): String
+  restartPatch(patchId: String!, abort: Boolean!, taskIds: [TaskToRestart!]!): String
```

Comment:
We should do this in a backwards compatible way. 
Lets create a new resolver call restartVersion() with a call signature that matches the pattern we want.

We can add the `@deprecated` directive to this field so we know we will be able to remove it soon.

Reply:
done